editor-plugins/vscode/src/Organizer.ts

```diff
@@ -264,18 +269,6 @@ export class Organizer {
 
     private put_lines_into(from: number, to: number, type: LineTypes) {
         let block: string[] = this.script_objects.slice(from, to);
-
-        for (let i = block.length - 1; i > 0; --i) {
-            let match_index = TYPE_MATCHES.findIndex(tm => block[i].match(tm));
-            if (match_index === -1) {
-                if (block[i].length === 0) {
-                    block.pop();
-                }
-            } else {
-                break;
-            }
-        }

```

Comment:
This loop was iterating over all the lines in a block and popping from the `block` array whenever an empty line was found. So, any newlines in the middle of the function would cause the end of the function to be discarded.

The broken code was passed into the formatter resulting in the issue reported.

https://github.com/Scony/godot-gdscript-toolkit/assets/46084870/797e7a73-fd29-44d3-b038-6baf32f84332

I'm sure it was added for a reason at the time, but it doesn't seem like the organizer should be concerning itself with newlines. That should be the formatter's responsibility.